src/algebra/group/pi.lean

```diff
@@ -133,8 +133,6 @@ variables {α β γ : Type*}
 
 @[simp, to_additive] lemma comp_one [has_one β] {f : β → γ} : f ∘ 1 = const α (f 1) := rfl
```

Comment:
I think this lemma and the one above probably should move to the other file too; they don't use the group structure at all.

Reply:
You could add a new section for `const_one` and put `const_mul` and `const_inv` in there too, probably not as `simp` lemmas.

Reply:
Do we agree that `pi.const_mul` and `pi.const_inv` do not yet exist?

Reply:
I didn't see them, but didn't look hard either